contents/tutorials/beta-feedback.md

```diff
@@ -0,0 +1,135 @@
+---
+title: How to collect beta feedback from beta users
+date: 2023-11-07
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+tags: ['surveys', 'feature flags']
+---
+
+The goal of a beta is to get a feature ready for a full release. This means fixing what isn’t working. Getting beta user feedback is one of the best ways to know what to fix, and PostHog makes it easy to collect.
```

Comment:
" This means fixing what isn’t working"

Theres a missing bit here on also getting feedback on what is working/what is useful etc. Its not only about what isnt working